mediaPanel.lua

```diff
@@ -41,33 +44,33 @@ function mediaPanel:SetSelectorInfo(button, mType, filePath, fileName)
 	button:Show()
 end
 
-function mediaPanel:Display(anchor, clicked, mediaType, get, set)
-	local mediaPanel = self:GetPanel()
-	local holding = mediaPanel.holding
+function Media:Display(anchor, clicked, mediaType, get, set)
+	local Media = self:GetPanel()
```

Comment:
I was just about to rewrite this file. Are you looking over it right now? I don't want to make any extreme changes if you are beginning to get familiar with it.